### PR TITLE
i18n(id): Update i18n.mdx translation

### PR DESCRIPTION
**Pull request by iyyafi**
#### Description
- Update ID translation guides for i18n.mdx file

### CODE REVIEW

**Review by HiDeoo — CHANGES_REQUESTED** [review]
Thanks for updating this translation :star2:

As a non-native speaker, I reviewed this PR using the [Docs Translation Review Checklist](https://hackmd.io/1KYnrpUQR0SAGTzHIUoUDg?view). I left one comment regarding some strings that should actually not be translated but otherwise, the translation looks good to me :thumbsup:

File: docs/src/content/docs/id/guides/i18n.mdx [hunk 1]
```diff
@@ -176,36 +214,20 @@ Anda dapat memberikan terjemahan untuk bahasa tambahan yang Anda dukung — atau
...
-   	"page.previousLink": "Previous",
-   	"page.nextLink": "Next",
-   	"404.text": "Page not found. Check the URL or try using the search bar.",
-   	"aside.note": "Note",
-   	"aside.tip": "Tip",
-   	"aside.caution": "Caution",
-   	"aside.danger": "Danger"
+   	"expressiveCode.copyButtonCopied": "Disalin!",
+   	"expressiveCode.copyButtonTooltip": "Salin ke papan klip",
+   	"expressiveCode.terminalWindowFallbackTitle": "Jendela terminal"
```
[the remark this answers is not in the document]
Comment (iyyafi) [reply docs/src/content/docs/id/guides/i18n.mdx]: Ah, i miss those. thank you for the review. I will fix it soon.

File: docs/src/content/docs/id/guides/i18n.mdx [hunk 2]
```diff
@@ -214,15 +236,168 @@ Anda dapat memberikan terjemahan untuk bahasa tambahan yang Anda dukung — atau
...
+   	"pagefind.clear_search": "Bersihkan",
+   	"pagefind.load_more": "Muat lebih banyak hasil",
+   	"pagefind.search_label": "Cari website ini",
+   	"pagefind.filters_label": "Filter",
+   	"pagefind.zero_results": "Tidak ada hasil untuk [SEARCH_TERM]",
+   	"pagefind.many_results": "[COUNT] hasil untuk [SEARCH_TERM]",
+   	"pagefind.one_result": "[COUNT] hasil untuk [SEARCH_TERM]",
+   	"pagefind.alt_search": "Tidak ada hasil untuk [SEARCH_TERM]. Menampilkan hasil untuk [DIFFERENT_TERM] sebagai gantinya",
+   	"pagefind.search_suggestion": "Tidak ada hasil untuk [SEARCH_TERM]. Coba salah satu pencarian berikut:",
+   	"pagefind.searching": "Mencari [SEARCH_TERM]..."
```
[the remark this answers is not in the document]
Comment (iyyafi) [reply docs/src/content/docs/id/guides/i18n.mdx]: Ok, will fix this soon too

**Review by HiDeoo — APPROVED** [review]
Thanks for the fixes, looks great to me now :star2:

**Pull request merged**